src/helpers/issue.ts

```diff
@@ -639,23 +639,37 @@ export const getOpenedPullRequestsForAnIssue = async (issueNumber: number, userN
   });
 };
 
-export const getOpenedPullRequests = async (username: string) => {
+export const getOpenedPullRequests = async (username: string, state: "ready" | "draft" | "all") => {
   const context = getBotContext();
   const prs = await getAllPullRequests(context, "open");
-  return prs.filter((pr) => !pr.draft && (pr.user?.login === username || !username));
+  return prs.filter((pr) => (state === "ready" ? !pr.draft : state === "draft" ? pr.draft : true) && (pr.user?.login === username || !username));
 };
 
 export const getCommitsOnPullRequest = async (pullNumber: number) => {
   const logger = getLogger();
   const context = getBotContext();
   const payload = getBotContext().payload as Payload;
   try {
-    const { data: commits } = await context.octokit.rest.pulls.listCommits({
-      owner: payload.repository.owner.login,
-      repo: payload.repository.name,
-      pull_number: pullNumber,
-    });
-    return commits;
+    const perPage = 100;
+    let curPage = 1;
+    const allCommits = [];
+    let fetchDone = false;
+    while (!fetchDone) {
+      const response = await context.octokit.rest.pulls.listCommits({
+        owner: payload.repository.owner.login,
+        repo: payload.repository.name,
+        pull_number: pullNumber,
+        per_page: 100,
+        page: curPage,
+      });
+      await checkRateLimitGit(response.headers);
```

Comment:
I am very concerned about the rate limit checker because we should definitely take consideration into the lambda function timeout (10 seconds). Anyone knows how long it is by the next reset time: `headers["x-ratelimit-reset"]`.

Reply:
Actually I found out that OctoKit has rate limit plugin installed by default so I think we don't even need this function.

Reply:
Please remove `checkRateLimitGit` from here @byteballet and mark it resolved once done.

Reply:
done